tests/Avalonia.Controls.UnitTests/VirtualizingStackPanelTests.cs

```diff
@@ -367,6 +367,38 @@ public void Focusing_Another_Element_Recycles_Original_Focus_Element()
             Assert.False(originalFocused.IsVisible);
         }
 
+        [Fact]
+        public void Focused_Element_Losing_Focus_Does_Not_Reset_Selection()
```

Comment:
The problem is actually in `SelectingItemsControl` but is exposed by a feature of `VirtualizingStackPanel`. It was easier to add a test here than in `SelectingItemsControlTests`.